src/components/dialogmote/TextareaField.tsx

```diff
@@ -0,0 +1,11 @@
+import { Textarea, TextareaProps } from "@navikt/ds-react";
+import React, { ForwardedRef } from "react";
+
+const TextareaField = (
```

Comment:
Bruker `forwardRef` her for at det skulle funke å bruke denne komponenten fra parenten med react-hook-form sin {...register()}. Bonus er at https://react-hook-form.com/docs/useform#shouldFocusError fungerer.